src/Microsoft.Sbom.Api/Executors/SBOMFileToFileInfoConverter.cs

```diff
@@ -21,7 +21,7 @@ public class SBOMFileToFileInfoConverter
     {
         private readonly IFileTypeUtils fileTypeUtils;
 
-        public SBOMFileToFileInfoConverter(FileTypeUtils fileTypeUtils)
+        public SBOMFileToFileInfoConverter(IFileTypeUtils fileTypeUtils)
```

Comment:
nit: since making change here already, maybe change `SBOMFileToFileInfoConverter` to `SbomFileToFileInfoConverter` for consistency in the codebase?